### PR TITLE
[DRAFT] Updates to data licensing / terms language

### PR DESCRIPTION
**Pull request by krivard**
This is a prerequisite for #286, but I'm handling it separately for easier reviewing.

This work includes:
* Revised language on the Epidata page in the licensing section
* A new COVIDcast-specific licensing page, to cover all license configurations and which COVIDcast sources they go with
* A more emphatic licensing section on the COVIDcast Epidata page
* A one-line specification of license for each COVIDcast source on its details page

Remaining work:
- [ ] GHT license
- [ ] link to terms of use wherever it will eventually be on the covidcast site

All data sources & licenses for all Epidata endpoints:
* COVIDcast DV,HA - HSP+Delphi, CC-BY
* COVIDcast GHT - Google, mirror (?)
* COVIDcast combo - Delphi, CC-BY
* COVIDcast JHU - JHU, mirror (?)
* COVIDcast Quidel - Quidel+Delphi, CC-BY
* COVIDcast Safegraph Mobility - Safegraph, mirror (?)
* COVIDcast Symptom Surveys - Delphi, CC-BY
* COVIDcast USAFacts - USAFacts, mirror (?)
* COVIDcast CHNG - Delphi, **CC-BY-NC**
* COVIDcast Safegraph Patterns - Safegraph+Delphi, CC-BY
* COVIDcast Google Symptoms - Google+Delphi, CC-BY
* COVIDcast NCHS - NCHS, mirror (?)
* AFHSB - no information - mirror?
* CDC - no information - mirror?
* Delphi Forecasts - no information - Delphi, CC-BY?
* Dengue Digital Surveillance - no information - mix of sources & licenses like COVIDcast?
* Dengue Nowcast - no information - Delphi, CC-BY?
* Digital Surveillance Sensors - no information - mix of sources & licenses like COVIDcast?
* ECDC ILI - mirror
* FluView - CDC, mirror
* FluView Clinical - no information - mirror?
* Flusurv - CDC, mirror
* Google Flu Trends - Google, mirror
* Google Health Trends - Delphi, CC-BY?
* ILI Nearby Nowcast - Delphi, CC-BY
* KCDC ILI - KCDC, mirror
* NIDSS Dengue - Taiwan CDC NIDSS, mirror
* NIDSS Flu - mirror?
* NoroSTAT - 
* PAHO Dengue -
* Quidel -
* Twitter Stream -
* Wikipedia Access -

### CODE REVIEW

File: docs/api/README.md [hunk 1]
```diff
@@ -37,17 +37,15 @@ like to cite it, we would gratefully recommend the following copy:
...
 ## Data licensing
 
-The majority of the data surfaced through this API is more or less just a
-carefully curated mirror of data acquired from various external parties. Such
-data is subject to its original licensing, where applicable.
-
----
+Several datasets surfaced through this API are carefully curated mirrors of data
+acquired from various external parties. Such data is subject to its original
+licensing, where applicable.
```
Comment (capnrefsmmat) [inline docs/api/README.md:42]: When #283 adds licensing details to all sources, I'd change this to something like "its original licensing, as noted in its source documentation page."

**Review by capnrefsmmat — APPROVED** [review]
looks fine to me

File: docs/api/README.md (hunk 1 above)
Comment (dfarrow0) [reply docs/api/README.md:42]: Sorry to be pedantic again. Just noting that #283 adds licensing details to `covidcast` sources only. The other Epidata source docs probably still need some work before we can remove this blanket licensing statement.

File: docs/api/covidcast.md [hunk 2]
```diff
@@ -41,6 +34,16 @@ href="https://lists.andrew.cmu.edu/mailman/listinfo/delphi-covidcast-api">subscr
 1. TOC
 {:toc}
 
+## Licensing
+Our CovidCast data is freely available under our [licensing](README.md#data-licensing)
+terms. For specific licensing on individual indicators we suggest users to visit [indicator licensing](covidcast_licensing.md).
```
Comment (Akvannortwick) [inline docs/api/covidcast.md:39]: The link is a place holder until such a file exists.

File: docs/api/covidcast.md [hunk 3]
```diff
@@ -41,6 +34,16 @@ href="https://lists.andrew.cmu.edu/mailman/listinfo/delphi-covidcast-api">subscr
 1. TOC
 {:toc}
 
+## Licensing
+Our CovidCast data is freely available under our [licensing](README.md#data-licensing)
+terms. For specific licensing on individual indicators we suggest users visit our [indicator licensing](covidcast_licensing.md) page.
```
Comment (Akvannortwick) [inline docs/api/covidcast.md:39]: Link is a place holder until such a file exists.

Comment (krivard) [inline docs/api/covidcast.md:39]:
```suggestion
Like all other Delphi Epidata datasets, the COVIDcast data is freely available to the public. Unlike other Delphi Epidata datasets however, our COVID-19 surveillance streams include data from many different sources, and data licensing is handled separately for each source. For a summary of the licenses used and a list of the indicators each license applies to,  we suggest users visit our [COVIDcast licensing](covidcast_licensing.md) page. Licensing information is also summarized on each indicator's details page.

```

Comment (Akvannortwick) [reply docs/api/covidcast.md:39]:
```suggestion
Like all other Delphi Epidata datasets, our COVIDcast data is freely available to the public. However, our COVID-19 surveillance streams include data from many different sources, with data licensing handled separately for each source. For a summary of the licenses used and a list of the indicators each license applies to, we suggest users visit our [COVIDcast licensing](covidcast_licensing.md) page. Licensing information is also summarized on each indicator's details page.
```

File: docs/api/README.md [hunk 4]
```diff
@@ -37,17 +37,15 @@ like to cite it, we would gratefully recommend the following copy:
...
 
 Any data which is produced novelly by Delphi and is intentionally and openly
 surfaced by Delphi through this API is hereby licensed
-[CC-BY](https://creativecommons.org/licenses/by/4.0/). Endpoints, as specified
-by the `source` parameter, which are known to wholly or partially serve data
-under this license include:
+[CC-BY](https://creativecommons.org/licenses/by/4.0/) except where otherwise
```
Comment (krivard) [inline docs/api/README.md:46]:
```suggestion
[CC BY](https://creativecommons.org/licenses/by/4.0/) except where otherwise
```

**Pull request merged**